src/it/scala/djinni/IntegrationTest.scala

```diff
@@ -97,12 +97,14 @@ class IntegrationTest extends FunSpec {
       cmd += s" --java-out $baseOutputPath/$idl/$JAVA"
       cmd += s" --jni-out $baseOutputPath/$idl/$JNI"
       cmd += s" --jni-header-out $baseOutputPath/$idl/$JNI_HEADERS"
+      cmd += " --jni-base-lib-include-prefix djinni/jni/"
```

Comment:
this should not be needed, we do not want to force users to specify include directories that should  just be found.